config/sample_application.yml

```diff
@@ -1,3 +1,6 @@
+REDIS_URL: "redis://localhost:6379"
```

Comment:
Added the Redis config as it was missing from the sample file.

Reply:
Weird, because this shouldn't be required, since both variables have defaults in `Envfile`, for example my `application.yml` does not contain any Redis conf

Reply:
Sounds good. Will try this again without my changes and see if all is good.

Reply:
So this works fine as you mention pulling out of `EnvFile`. I think I know what happened. When I didn't have Redis installed, I got connection errors and so I added it to application.yml. After I installed Redis things worked, so just me thinking two things fixed the situation.